src/components/date/__internal__/navbar/navbar.test.tsx

```diff
@@ -0,0 +1,193 @@
+  await user.keyboard("{arrowleft}");
+
+  expect(onPreviousClick).not.toHaveBeenCalled();
+});
+
+test("should not change the current month when 'arrowright' key is pressed and previous button is focused", async () => {
+  const user = userEvent.setup();
+  const onPreviousClick = jest.fn();
+  render(<Navbar onPreviousClick={onPreviousClick} onNextClick={jest.fn()} />);
+  const prevButton = screen.getByRole("button", { name: "Previous month" });
+  prevButton.focus();
+  await user.keyboard("{arrowright}");
+
+  expect(onPreviousClick).not.toHaveBeenCalled();
+});
+
+test("should not change the current month when 'arrowdown' key is pressed and next button is focused", async () => {
+  const user = userEvent.setup();
+  const onNextClick = jest.fn();
+  render(<Navbar onPreviousClick={jest.fn()} onNextClick={onNextClick} />);
+  const nextButton = screen.getByRole("button", { name: "Next month" });
+  nextButton.focus();
+  await user.keyboard("{arrowdown}");
+
+  expect(onNextClick).not.toHaveBeenCalled();
+});
+
+test("should not change the current month when 'arrowup' key is pressed and next button is focused", async () => {
+  const user = userEvent.setup();
+  const onNextClick = jest.fn();
+  render(<Navbar onPreviousClick={jest.fn()} onNextClick={onNextClick} />);
+  const nextButton = screen.getByRole("button", { name: "Next month" });
+  nextButton.focus();
+  await user.keyboard("{arrowup}");
+
+  expect(onNextClick).not.toHaveBeenCalled();
+});
+
+test("should not change the current month when 'arrowleft' key is pressed and next button is focused", async () => {
+  const user = userEvent.setup();
+  const onNextClick = jest.fn();
+  render(<Navbar onPreviousClick={jest.fn()} onNextClick={onNextClick} />);
+  const nextButton = screen.getByRole("button", { name: "Next month" });
+  nextButton.focus();
+  await user.keyboard("{arrowleft}");
+
+  expect(onNextClick).not.toHaveBeenCalled();
+});
+
+test("should not change the current month when 'arrowright' key is pressed and next button is focused", async () => {
+  const user = userEvent.setup();
+  const onNextClick = jest.fn();
+  render(<Navbar onPreviousClick={jest.fn()} onNextClick={onNextClick} />);
+  const nextButton = screen.getByRole("button", { name: "Next month" });
+  nextButton.focus();
+  await user.keyboard("{arrowright}");
+
+  expect(onNextClick).not.toHaveBeenCalled();
+});
+
+test("should change the current month when 'Enter' key is pressed and previous button is focused", async () => {
+  const user = userEvent.setup();
+  const onPreviousClick = jest.fn();
+
+  render(<Navbar onPreviousClick={onPreviousClick} onNextClick={jest.fn()} />);
+  const prevButton = screen.getByRole("button", { name: "Previous month" });
+  prevButton.focus();
+  await user.keyboard("{Enter}");
+
+  expect(onPreviousClick).toHaveBeenCalled();
+});
+
+test("should change the current month when ' ' (Space) key is pressed and previous button is focused", async () => {
```

Comment:
suggestion(minor, non-blocking): just put space rather than ' '

The enter tests just say enter, so I don't see a reason why we couldn't just say space. Very minor, barely worth the effort of changing at all though.